### ISSUE TITLE
What means 'the resulting model is quantization aware but not quantized'

### ISSUE THREAD

**Issue by wptkddms** [tensorflow/tensorflow#42250]
Can anybody explain me what means 'the resulting model is quantization aware but not quantized' on [tutorial](https://www.tensorflow.org/model_optimization/guide/quantization/training_example?hl=en)
Is there no way to use Quantization-Aware-Training without tflite?

Thank you.

**Comment by Saduf2019** [comment]
@wptkddms 
Where in the tutorial does it say that the model is quantization aware but not quantized?

QAT is done [model is typically trained ] to compensate for the loss in precision that might be introduced due to quantization. (When you reduce the precision of the parameters of your model, it can result in information loss and you might see some reduction in the accuracy of your model. In these situations, quantization-aware training can be really helpful.)

**Comment by wptkddms** [comment]
@Saduf2019 
Thank you for reply

[here](https://www.tensorflow.org/model_optimization/guide/quantization/training_example?hl=en#define_the_model) is the sentence.
Using the tutorial, I printed the weights of q_aware_model, but the data type was float32, not int8. So I'm confused.

**Comment by Saduf2019** [comment]
@wptkddms P
Please refer to [this link](https://blog.tensorflow.org/2020/04/quantization-aware-training-with-tensorflow-model-optimization-toolkit.html)
